R/mcmc.R

```diff
@@ -396,24 +396,22 @@ setMethod("mcmc",
                      ...){
 
               ## decide whether we sample from the prior or not
-              from_prior <- data@nObs == 0L
+              from_prior <- data@nObs == 0L # TODO shouldn't be no DLT only?
```

Comment:
maybe check with @Winnie-Yeung

Reply:
Sure, Tx!

Reply:
Winnie does not really remember about how this parameter can be used in this function, so I just removed it.